src/services/transactions/sendBatchTransactions.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  BatchTransactionsResponseType,
+  SendBatchTransactionReturnType,
+  SignedTransactionType
+} from 'types';
+import axios from 'axios';
+import { TRANSACTIONS_BATCH } from 'apiCalls';
+
+export interface SendBatchTransactionsPropsType {
+  transactions: SignedTransactionType[] | SignedTransactionType[][];
+  account: string;
+  sessionId: string;
+  apiAddress: string;
+}
+
+export async function sendBatchTransactions({
+  transactions,
+  sessionId,
+  account,
+  apiAddress
+}: SendBatchTransactionsPropsType): Promise<SendBatchTransactionReturnType> {
+  try {
+    const batchId = `${sessionId}-${account}`;
```

Comment:
why include account here since sessionId is already a timestamp and you cannot log in with 2 accounts at the same time?

Reply:
There is an API convention. You don't have to be logged in to track the batches transactions of an account